bots/tests-invoke

```diff
@@ -183,7 +181,7 @@ class PullTask(object):
             # explicit request for html logs
             status["link"] = "log.html"
             status["extras"] = [ "https://raw.githubusercontent.com/cockpit-project/cockpit/master/bots/task/log.html" ]
-        elif self.context.startswith("cockpit/"):
+        elif self.test_project:
```

Comment:
This will regress the log.html bits for cockpit further down below, right? At least we want test_project to be something like `cockpit-project/cockpit` for cockpit itself, or isn't that the case yet?

Reply:
Right, I also though that `TEST_PROJECT` would be set up to `cockpit-project/cockpit` but it isn't.
This is what `tests-scan` outputs for cockpit test:
```
PRIORITY=0001 bots/make-checkout --verbose --base=master --repo=cockpit-project/cockpit pull/12361/head 0985a3bbfff2a97fa7f36d1a9253280db9dd3202 && GITHUB_BASE=cockpit-project/cockpit TEST_NAME=pull-12361-20190719-132222 TEST_REVISION=0985a3bbfff2a97fa7f36d1a9253280db9dd3202 bots/tests-invoke --pull-number=12361  --rebase=master fedora-30/selenium-chrome pull/12361/head
```
and this is for external project:
```
PRIORITY=0005 bots/make-checkout --verbose --base=master --repo=cockpit-project/starter-kit --bots-ref=pull/12361/head '' && TEST_PROJECT=cockpit-project/starter-kit TEST_NAME=pull-12361-20190719-132222 TEST_REVISION=0985a3bbfff2a97fa7f36d1a9253280db9dd3202 bots/tests-invoke --pull-number=12361  --rebase=master --remote=test centos-7 ''
```
I admit that I took this shortcut and just misused the fact that this is not set up.

Reply:
Right, I figured something like that -- too many "cockpit" special cases. While we are at it, could you please test for cockpit here, and somewhere at the top make it default to cockpit, with a FIXME?